scripts/cfos/make_dataset_data_tables.py

```diff
@@ -87,29 +88,34 @@ def generate_load_table_files(dataset_tables_dict):
     return tsv_filenames_list
 
 
-def create_dataset_tables_dictionary(validated_dataset, dataset_name, dataset_table_names, schema_dict):
-    """For a given dataset, subset/separate input columns into individual table dataframes."""
-
-    # instantiate empty dictionary to capture {table_name: table_metadata_dataframe}
-    dataset_tables_dict = {}
-
-    # create a subset df for each table defined in the chosen dataset
-    for table in dataset_table_names:
-        # get subset dataframe for specific table for given dataset name
-        table_df = dataset_metadata[schema_dict[dataset_name][table]]
-
-        # TODO: edit the value = dataframe for a table to have the entity:table_name_id format
-
-        # add KVP {table name : table_df} entry to all tables dictionary
-        dataset_tables_dict[table] = table_df
-
-    print(f"Success: Dataset has been partitioned into {len(dataset_table_names)} tables: {dataset_table_names}")
-    return dataset_tables_dict
-
-
-def validate_dataset(dataset):
+def validate_dataset(dataset, schema_dict, field_dict):
+    for table in list(schema_dict.keys()):
```

Comment:
Should this be validating the columns vis a vis entire dataset only?

Reply:
What do you mean by vis a vis entire dataset?

Reply:
Does this function validate columns or columns and the data too?